Web/starter_project_vanguard/components/common/Loading.tsx

```diff
@@ -0,0 +1,33 @@
+export const LoadingSpinner = (props: { size?: number }) => {
+    return (
+      <div role="status">
```

Comment:
Rather than creating a custom spinner, It is good to use icons like `AiOutlineLoading3Quarters` with the `animate-spin` class. check this out [link](https://tailwindcss.com/docs/animation)